diff --git a/lib/TransformNormalModulePlugin.js b/lib/TransformNormalModulePlugin.js
index 5b2a53dcb559071d33ba9bfd93da191801a13c43..e91a7a580cb717944892e2f755ff2a0fa3dd6a01 100644
--- a/lib/TransformNormalModulePlugin.js
+++ b/lib/TransformNormalModulePlugin.js
@@ -598,7 +598,7 @@ class TransformNormalModulePlugin {
           module.request &&
           (cacheable(module) || !module.built) &&
           module instanceof NormalModule &&
-          (!frozen ||
+          (!frozen || !frozen.build ||
             (schema >= 4 && module.hash !== frozen.build.hash) ||
             (schema < 4 &&
               module.getHashDigest(extra.compilation.dependencyTemplates) !==
