diff --git a/lib/watchpack.js b/lib/watchpack.js
index ddc9138c2ea369334768bae1ce6722b3256fe091..48bcb3af554e39657006c1a776b06030790d1d67 100644
--- a/lib/watchpack.js
+++ b/lib/watchpack.js
@@ -49,7 +49,7 @@ const ignoredToFunction = ignored => {
 
 const normalizeOptions = options => {
 	return {
-		followSymlinks: !!options.followSymlinks,
+		followSymlinks: true,
 		ignored: ignoredToFunction(options.ignored),
 		poll: options.poll
 	};
